### PR TITLE
New Floorsigns

### PR DESCRIPTION
**Pull request by lngredients**
Mostly for medbay. Useful?

### CODE REVIEW

**Comment by bloxgate** [comment]
Why does this remove all your previous changelogs?

**Review by caelaislinn — CHANGES_REQUESTED** [review]
Restore the missing changelogs please

**Comment by lngredients** [comment]
Added them back in.

**Comment by lngredients** [comment]
That should do it.

File: html/changelogs/Ingredients New Floor signs 12 July.yml [hunk 1]
```diff
@@ -33,8 +33,4 @@ delete-after: True
 # Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
 # Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
 changes: 
-  - tweak: "Added in random clothes sets on Innie ship. Some random materials as well."
-  - tweak: "Random weapon and ammo spawns were added in as well to the innie ship."
-  - tweak: "A few static goodies were spread around the ship as well."
-  - tweak: "Added in two new types of armors, the Colossus Armor and the Zeal Scout Suit. Made a case which can carry the colossus armor."
-  - maptweak: "Added in a new room to house the two new armor types."
\ No newline at end of file
+  - imageadd: "New floorsigns for medical."
```
Comment (caelaislinn) [inline html/changelogs/Ingredients New Floor signs 12 July.yml]: By the looks of it you have still deleted a changelog file "Ingredients URF Ship Items and New Innie Armors 30 June.yml"

**Review by caelaislinn — DISMISSED** [review]

File: html/changelogs/Ingredients New Floor signs 12 July.yml (hunk 1 above)
Comment (lngredients) [reply html/changelogs/Ingredients New Floor signs 12 July.yml]: Found it and added it in.

**Review by BDpuffy420 — APPROVED** [review]

**Pull request merged**